### ISSUE TITLE
In fullscreen mode with minutes-only timer, it is hard to judge if the timer is running after app start

### ISSUE THREAD

**Issue by reox** [adrcotfas/goodtime#290]
I use fullscreen mode in combination with the minutes-only timer design. However, when I open the app, it is hard to judge if the timer is actually running or currently paused (which is the case after opening the app), because it does not move and you can not see the notification bar to check whenever the phone is in DND mode.
If I then press the timer twice to go into the real pause mode, it blinks - which is a clear indication of pause.
For the initial display though, it would be nice to see that the timer is not running - maybe indicate that with a little symbol or text below the timer, that the session has to be started.

**Comment by adrcotfas** [comment]
I agree and think that an extra indicator would help (will keep this in mind for the UI improvements which will come in the near future).

**Comment by adrcotfas** [comment]
This was addressed and will come in the next release.
